Database/Corrections/classicQuestFixes.lua

```diff
@@ -15,7 +15,7 @@ local l10n = QuestieLoader:ImportModule("l10n")
 -- Further information on how to use this can be found at the wiki
 -- https://github.com/Questie/Questie/wiki/Corrections
 
-function QuestieQuestFixes:Load()
+function QuestieQuestFixes:LoadMissingQuests()
```

Comment:
I separated this, because the order i now want is:

1. add the missing indices (`QuestieQuestFixes:LoadMissingQuests()`)
2. load reputation fixes from wowhead
3. load corrections (`QuestieQuestFixes:Load()`)

This is so that the manual corrections can still overwrite the automated ones

Reply:
This is a great idea. Even though I am not sure if we even still need `LoadMissingQuests` 🤔